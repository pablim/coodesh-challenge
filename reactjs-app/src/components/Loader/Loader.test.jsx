import { render, screen, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Loader } from '../'

describe('Reders tests', () => {
	test('Render Loader component', () => {
		const { container } = render(<Loader />);
		expect(container).toMatchSnapshot();
	});
})
