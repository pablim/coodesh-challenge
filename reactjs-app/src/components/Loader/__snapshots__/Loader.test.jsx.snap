// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reders tests Render Loader component 1`] = `
<div>
  <div
    class=""
  >
    carregando...
  </div>
</div>
`;
